package.json

```diff
@@ -6,6 +6,8 @@
   "license": "Apache 2.0",
   "repository": "https://github.com/oppiabot/oppiabot.git",
   "scripts": {
+    "dev": "nodemon",
```

Comment:
is this a dependency for updated probot?

Reply:
No.
It's a dev dependency. Used to re-run the app when files change.